index.html

```diff
@@ -25,16 +25,18 @@ <h1>Hacker Escape Rooms</h1>
 
     <nav class="nav__large">
         <ul>
+            <a href="index.html">
+                <li class="Online">Home</li>  <!--I noticed there was no link to home page (If you go to filter no way to go back to home page )-->
```

Comment:
As you wish guys i can remove it . its just if i am in filter page then i am stuck there and have to exit the site and enter it again to go to home page .

Reply:
Yes, I agree with you.  This is at home page so that is why I was wondering if it is necessary here too. Just a thought. It can be here also if you like :)